pipeline/terraform/test-environment/test-environment.tf

```diff
@@ -0,0 +1,63 @@
+provider "aws" {
+  profile = "default"
+  region = var.region
+}
+
+resource "aws_ecs_task_definition" "test-environment-mhs-task" {
+  family = "${var.build_id}-mhs-task"
```

Comment:
Optional/v. minor: could look a bit nicer with the build id at the end instead of at the beginning.

Reply:
Moved